tests/topotests/lib/bgp.py

```diff
@@ -60,6 +62,15 @@ def create_router_bgp(tgen, topo, input_dict=None, build=False):
             "bgp": {
                 "local_as": "200",
                 "router_id": "22.22.22.22",
+                "graceful-restart": {
+                    "graceful-restart": True,
+                    "preserve-fw-state": True,
+                    "timer": {
+                        "restart-time": 300,
```

Comment:
shoudn't these lower timers reduce tests time?

Reply:
This example data in framework API. It will guide users, how to call create_router_bgp(). This data will not be used anywhere.